object.js

```diff
@@ -143,9 +143,17 @@ function hint(gj, options) {
                     });
                 }
             }
-            coords.forEach(function(c) {
-                positionArray(c, type, depth - 1, c.__line__ || line);
-            });
+            else if (!Array.isArray(coords)) {
```

Comment:
plz follow existing code style and `} else if {` instead of a linebreak between `}` and `else if`